torch/testing/_internal/common_quantization.py

```diff
@@ -252,9 +252,9 @@ def forward(self, x):
         return x
 
 class SingleLayerLinearDynamicModel(torch.nn.Module):
-    def __init__(self):
+    def __init__(self, qengine='fbgemm'):
```

Comment:
We normally use @override_qengines for sweeping over qengines. Please do not pass qengine as a parameter to the model, instead in the test, specify the qengine using the @override_qengines